### ISSUE TITLE
Perseus tool in geometry section not working

### ISSUE THREAD

**Issue by rifatbd** [learningequality/ka-lite#5099]
## Summary
In the geometry section (more specifically in Transformations-Congruence-Similarity section) when doing exercise no operation can be done on shapes. Like rotating the shape, or transform or operations like those.

## Branch or installer method

 - Installer: Windows installer from pantry (win7 64 bit)
 - Version: 0.16.2 

## Traceback or relevant snippet from server.log

server.log is empty

## How to reproduce

1. Start Ka lite & naviagte to /learn/khan/math/basic-geo/transformations-congruence-similarity-geo/transformations-basics/performing-reflections-on-the-coordinate-plane/
2. Try to do the "Reflect" operation by clicking the button underneath the image
3. Nothing happens

## Screenshots

![capture1](https://cloud.githubusercontent.com/assets/18570759/14703090/cc4fe4b8-07cf-11e6-99d8-0279a2e03d65.PNG)
![capture2](https://cloud.githubusercontent.com/assets/18570759/14703089/cc4f17f4-07cf-11e6-80ad-0070dba9ef7a.PNG)
![capture3](https://cloud.githubusercontent.com/assets/18570759/14703093/cc50ab96-07cf-11e6-8486-e5b4cc12ad4c.PNG)
![capture4](https://cloud.githubusercontent.com/assets/18570759/14703091/cc500f9c-07cf-11e6-81f9-1528a86d7e01.PNG)
![capture5](https://cloud.githubusercontent.com/assets/18570759/14703092/cc504fac-07cf-11e6-8eb9-422544bba589.PNG)

**Comment by benjaoming** [comment]
@rtibbles what's your take on this? You mentioned as a possible dirty resolution to #5036 that exercises could be excluded.

**Comment by benjaoming** [comment]
I find that lots of geometry exercises are broken, almost anyone that I click. I guess these are using the "more advanced" features of Perseus.

Using: Latest English content pack + 0.16.3 release.

Here's another example of an error:

![screenshot from 2016-04-26 16 07 27](https://cloud.githubusercontent.com/assets/374612/14821064/99a7daea-0bc9-11e6-9230-7db02447a7f0.png)

**Comment by rifatbd** [comment]
@benjaoming how can i install the 0.16.3 release? I cant see any installer for windows. Or there is some other method.

**Comment by benjaoming** [comment]
@rifatbd 0.16.3 doesn't solve this issue, it was released in our PPA and on PyPi but will quickly be made redundant because fixing the above is of utmost priority :)

**Comment by benjaoming** [comment]
@rifatbd thanks for a very thorough issue report btw, made it very easy to realize the problems with exercises.

**Comment by rifatbd** [comment]
Good to hear that i was of some help...... :)

**Comment by mjptak** [comment]
also glad to see that the very dot that lowers the exercise creation bar is
still intact.  Awesome work...awesome applications...awesome volunteers.

**Comment by benjaoming** [comment]
It seems like everything will be fine in the next release, we just need to re-generate some of the exercise data before releasing 0.16.4. Within another 24 hours it'll be rolled out :)

**Comment by rifatbd** [comment]
@benjaoming I just upgraded to 0.16.4 from PyPi .. and the issue seems to be solved :)....
One more thing the woff2 fonts are still missing in fresh installation for some reason..... i just had to manually add them.....may b u should check that out too..... 
apart from that the very issue of geometric tools is solved :+1: ........ 
i added a snap of woff2 missing just in case..
![capture22](https://cloud.githubusercontent.com/assets/18570759/14935313/5987ffda-0eee-11e6-8d95-0720786d7ae8.PNG)

**Comment by rifatbd** [comment]
Anyway looks like there is still some minor issues left .....
When Bengali is selected as Default language this happens....

![capture21](https://cloud.githubusercontent.com/assets/18570759/14935445/b344ed26-0ef3-11e6-960d-692f3b139c67.PNG)
![capture23](https://cloud.githubusercontent.com/assets/18570759/14935446/b34884c2-0ef3-11e6-8adc-bb6cbcdaa89e.PNG)

**Comment by benjaoming** [comment]
Thanks for testing @rifatbd !!

It seems like said fonts are located in `static/html/_static/fonts/`.

I would recommend creating two new issues, one for issues regarding Bengali, the other for fonts. I can tag the latter as "beginner friendly". It can be addressed possibly in 0.16.5.

**Comment by benjaoming** [comment]
Sorry, the fonts are present in the right location, they are however not included in MANIFEST.in

```
➜  ka-lite git:(0.16.x) find |grep KaTeX_Main-Regular.woff2   
./kalite/distributed/static/fonts/KaTeX_Main-Regular.woff2
./kalite/distributed/static/js/distributed/perseus/ke/local-only/katex/fonts/KaTeX_Main-Regular.woff2
./kalite/static/fonts/KaTeX_Main-Regular.woff2
./kalite/static/js/distributed/perseus/ke/local-only/katex/fonts/KaTeX_Main-Regular.woff2
```

**Comment by benjaoming** [comment]
Gonna test now manually if this works, will give an update and release as 0.16.5 soon.

**Comment by benjaoming** [comment]
Font issue solved in 0.16.5
